docs/teamshares/recipes/buttons.md

```diff
@@ -0,0 +1,190 @@
+# How to convert SharedUI::Button usages to Shoelace
+
+## 1. Link button
+
+> 💡 Hint: these are buttons marked with `external: true` or marked with `render_as_link: true`
```

Comment:
I like these little blockquote hints. The italicized emoji looks a bit weird, but we can figure out formatting for that later.

I think some of the hints here should just be paragraphs of explanation, in which I'd err on the side of over-explaining. Here, I would just make this a paragraph that says "`SharedUI::ButtonComponent`s with `external: true` or `render_as_link: true` are rendered as HTML `<a>` tags. To get a similar effect in Shoelace, use `sl-button` with an `href` attribute."